code/game/objects/items/weapons/dice.dm

```diff
@@ -122,3 +122,172 @@
 				for(var/obj/machinery/computer/bhangmeter/bhangmeter in doppler_arrays)
 					if(bhangmeter)
 						bhangmeter.sense_explosion(epicenter.x,epicenter.y,epicenter.z,round(uncapped*0.25), round(uncapped*0.5), round(uncapped),"???", cap)
+
+
+/obj/item/weapon/dice/d20/cursed
+	desc = "Something about this dice seems wrong"
+	name = "/improper Mysterious d20"
+	var/deactivated = 0 //Eventually the dice runs out of power
+
+/obj/item/weapon/dice/d20/cursed/pickup(mob/user as mob)
+	if(deactivated == 0)
+		to_chat(user, "<span class=sinister><B>Are you feeling lucky?</span></B>")
+	else
+		return 0

```

Comment:
else return 0 could be removed which I intend to do, just forgot last night.

The dice deactivates itself after a few rolls and becomes essentially a normal dice (though a badmin can re-activate it through variable editing).